tests/HUBComponentWrapperTests.m

```diff
@@ -101,18 +169,19 @@ - (HUBComponentWrapper *)componentWrapperForComponent:(id<HUBComponent>)componen
 
 #pragma mark - HUBComponentWrapperDelegate
 
-// Below methods are all no-ops implemented out of necessity.
-
 - (void)componentWrapper:(HUBComponentWrapper *)componentWrapper willUpdateSelectionState:(HUBComponentSelectionState)selectionState
 {
+    self.selectionStateFromWillUpdateDelegateMethod = selectionState;
 }
 
 - (void)componentWrapper:(HUBComponentWrapper *)componentWrapper didUpdateSelectionState:(HUBComponentSelectionState)selectionState
 {
+    self.selectionStateFromDidUpdateDelegateMethod = selectionState;
 }
 
 - (void)componentWrapper:(HUBComponentWrapper *)componentWrapper childSelectedAtIndex:(NSUInteger)childIndex
 {
+    // No-op
```

Comment:
"No operation". That this method is implemented out of neccessity to conform to a protocol, not because we need it :)